browser/ui/webui/brave_rewards/rewards_page_handler.cc

```diff
@@ -418,7 +413,7 @@ void RewardsPageHandler::GetAdsSettings(GetAdsSettingsCallback callback) {
       prefs_->GetBoolean(brave_ads::prefs::kShouldAllowSubdivisionTargeting);
 
   auto& subdivisions = brave_ads::GetSupportedSubdivisions();
-  auto iter = subdivisions.find(brave_l10n::GetCountryCode(GetLocalState()));
+  auto iter = subdivisions.find(brave_l10n::GetDefaultISOCountryCodeString());
```

Comment:
It’s correct that we use the device region to determine whether to show ads settings. There was a bug where Rewards users were shown settings based on GeoIP instead, which will be resolved by https://github.com/brave/brave-core/pull/29666.

Reply:
It’s correct that we use the device region to determine whether to show ads settings. There was a bug where Rewards users were shown settings based on GeoIP instead, which will be resolved by https://github.com/brave/brave-core/pull/29666. A follow-up will transition to using Chromium APIs for country code, and brave_l10n will continue to be deprecated.